rpm/xcache.spec

```
@@ -1,6 +1,6 @@
 Name:      xcache
 Summary:   XCache scripts and configurations
-Version:   1.0.5
+Version:   1.0.6
```

Let's bump the version

```suggestion
Version:   1.1.0
```